### ISSUE TITLE
Power by lipo

### ISSUE THREAD

**Issue by Andi68-at** [danjulio/lepton#9]
Hello 
Can I use the Tcam-mini directly with a 3.7V lipo or do I need a voltage converter?
how many mAh
thanks

**Comment by danjulio** [comment]
Yes, it can be powered directly by a LiPo connected to the 5V terminals.  It requires 250-300 mA operating current.  Two things to consider:

1. Never connect USB power when the LiPo is attached.
2. tCam-Mini will fail when the Lipo voltage falls near or below 3.3V as the regulator providing 3.3V to the ESP32 will drop out of regulation.  You should provide a low-voltage shutdown circuit if possible set to about 3.3 - 3.4V.  This still provides you most of the energy in the LiPo.
